perun-core/src/main/java/cz/metacentrum/perun/core/bl/AuditMessagesManagerBl.java

```diff
@@ -1,17 +1,14 @@
 package cz.metacentrum.perun.core.bl;
```

Comment:
I guess not, just remove any listings like: ``( perunadmin | voadmin | groupadmin | self | facilityadmin | voobserver | topgroupcreator )`` and other stuff I pointed out. Thank you.